src/main/java/games/strategy/engine/framework/ui/SaveGameFileChooser.java

```diff
@@ -32,17 +31,6 @@ public static String getAutoSaveFileName() {
     return AUTOSAVE_FILE_NAME;
   }
 
-  public static String getAutoSave2FileName() {
```

Comment:
This removal broke looks to have broke version compat:
```
java.lang.NoSuchMethodError: games.strategy.engine.framework.ui.SaveGameFileChooser.getAutoSave2FileName()Ljava/lang/String;
	at games.strategy.engine.framework.ServerGame.autoSave(ServerGame.java:354)
	at games.strategy.engine.framework.ServerGame.runStep(ServerGame.java:438)
	at games.strategy.engine.framework.ServerGame.startGame(ServerGame.java:271)
	at games.strategy.engine.framework.startup.launcher.ServerLauncher$1.run(ServerLauncher.java:203)
```
interesting it is a static method, I thought those were typically safe to remove.